modules/knative-serving-advanced-config.adoc

```diff
@@ -15,17 +14,16 @@ Do not modify any YAML contained inside the `config` field. Some of the configur
 
 image::serving-form-view.png[Create Knative Serving form in web console Administrator Perspective]
 
-[id="knative-serving-controller-custom-certs_{context}"]
```

Comment:
Is there a reason that you're removing these? I do think that headings (even those below a level 1) should have an ID set with the context. Because otherwise it might cause a duplicate ID error if the module is used in more than 1 assembly.

Same w/ the rest of the IDs removed throughout the PR